Sources/POSIXCore/POSIXError.swift

```diff
@@ -3,30 +3,34 @@
 
 #if os(macOS) || os(Linux)
 
-#if os(macOS)
-import Darwin
-#elseif os(Linux)
-import Glibc
-#endif
+  #if os(macOS)
+    import Darwin
+  #elseif os(Linux)
+    import Glibc
+  #endif
 
-public struct POSIXError: Error {
-  internal let code: CInt
+  public struct POSIXError: Error {
+    internal let code: CInt
 
-  public init(_ code: CInt = errno) {
-    self.code = code
+    public init(_ code: CInt = errno) {
+      self.code = code
+    }
   }
-}
 
-extension POSIXError: CustomStringConvertible {
-  public var description: String {
-    return withUnsafeTemporaryAllocation(of: CChar.self, capacity: 256) {
-      guard strerror_r(code, $0.baseAddress, $0.count) == 0,
-          let baseAddress = $0.baseAddress else {
-        return "POSIX Error \(code)"
+  extension POSIXError: CustomStringConvertible {
+    public var description: String {
+      return withUnsafeTemporaryAllocation(of: CChar.self, capacity: 256) {
+        guard let _baseAddres = $0.baseAddress else {
```

Comment:
I think that this will break on macOS.

Reply:
Ugh, never mind, we already unwrap afterwards. Okay, hoisting the `baseAddress` makes sense.